roles/pmm/defaults/main.yaml

```diff
@@ -22,9 +22,28 @@ pmm_agent_setup_flags:
   server-password: '{{ pmm_agent_token }}'
   server-insecure-tls: '{{ pmm_server_insecure }}'
 
-pmm_ms_folder: 'ManagedService'
-pmm_alertmanager_contactpoint_uid: managedservice_contact_point
+pmm_template_folder: '{{ pmm_default_template_folder | default("DefaultFolder")}}'
+pmm_alertmanager_contactpoint_uid: '{{ pmm_alertamanger_default_contactpoint_uid | default("DefaultContact_")}}'
+pmm_templates_prefix: '{{ pmm_default_templates_prefix | default("DefaultPrefix_")}}'
+pmm_alerting_runbook_path: 'https://localhost/runbooks/'              # path used in alert labels
+pmm_alertmanager_rule_periods:                                                                          # Used to set the "for" duration in rules
+  interval_unit: m
+  low_interval: 1
+  med_interval: 5
+  high_interval: 10
+  max_interval: 15
+pmm_alert_filters: []            # example: '{"VmalertAlertsError":{"filters":[{"label":"env","regexp":"staging","type":"MATCH"},{"label":"node_name","regexp":"db10","type":"MISMATCH"}]}}'
 
+## Alert Manager Rules Thresholds
+pmm_alertmanager_thr_PostgresqlCommitRateLow: 80
```

Comment:
This should be defined a different way, it means that there is going to need to be a value for anything with a threshold.
I don't see why these need to be present as the values would be set when creating the alert rule and so it doesn't matter what is set in the template.

Reply:
No, this is set in the template not in the alert rule. In the alert rule you do not change the expression.

Reply:
So, how are rules being created from a single template with 2 different thresholds?

Reply:
that's two different template.  You can not edit the expression even on GUI when you are creating an alert.

Reply:
Now I have added `params` to all the templates, thresholds can be overwritten now from inventory but the syntax is different for alert with thresholds and without them.  The API is not smart enough to use the default values from the template you have to redefine them  again when you create the alert.. anyhow now this feature is added. will push it soon.